apidef/oas/server.go

```diff
@@ -11,6 +11,8 @@ type Server struct {
 	Slug string `bson:"slug,omitempty" json:"slug,omitempty"`
 	// Authentication contains the configurations related to authentication to the API.
 	Authentication *Authentication `bson:"authentication,omitempty" json:"authentication,omitempty"`
+	// ClientCertificates contains configuration related to certificates are they enabled and list of enabled certificates
```

Comment:
`ClientCertificates contains the configurations related to static mTLS.` It would be better and let's not forget `.` at the end for consistency